src/Raven.Client/Documents/Operations/Replication/PullReplicationAsSink.cs

```diff
@@ -78,6 +78,22 @@ public override DynamicJsonValue ToJson()
             return djv;
         }
 
+        public DynamicJsonValue ToAuditJson()
+        {
+            if (string.IsNullOrEmpty(HubName))
```

Comment:
no need.  this condition has slipped in by mistake. better it would be to removed it, since it's not the logging action responsibility.  (btw the ctor will throw earlier )